packages/suite/src/views/wallet/send/components/Outputs/components/Amount/index.tsx

```diff
@@ -246,7 +261,7 @@ export const Amount = ({ output, outputId }: AmountProps) => {
                                 )}
                             </Label>
                         }
-                        bottomText={bottomText}
+                        bottomText={bottomText || null}
```

Comment:
Why do you added `|| null`?

Reply:
https://github.com/trezor/trezor-suite/blob/04a0e2ce8aeeceb8377b5a8425089b24b835c35c/packages/components/src/components/form/Input/Input.tsx#L100